todo.md

```diff
@@ -32,32 +32,29 @@ registration, and by outreach through education.
 
 ### Abstract/Proposal Summary (required) maximum of 250 words
 
-Our proposal has two major components:
-
-* Strengthening code foundations to make Nibabel more useful as an *brain
-  image workbench* and *base library*.
-* Broadening social foundations by building a platform and content for
-  teaching neuroimaging with Nibabel.
-
-We will *strengthen code foundations* by:
-
-* Expanding the API to support image metadata by adding axis and tick labels.
-* Extending support for reading and storing metadata from scanner outputs.
-* Extending output formats to store metadata.
-* Designing a standard API for brain surface data.
-* Adding support for loading, manipulating, applying and storing outputs from
-  image registration.
-
-We will *broaden social foundations* by:
-
-* Creating an open-access online course to teach neuroimaging using Nibabel as
-  brain image workbench, built on existing materials from our neuroimaging teaching in Berkeley.
-* Using the course materials as open, engaging and comprehensive documentation
-  for the library.
-* Recruiting a broader range of developers with targeted scholarships in the
-  form of paid course certificates.
-* Building and documenting the platform for delivering the course so that other
-  fields and libraries can build on this foundation.
+Nurturing and engaging a diverse community around base libraries like Nibabel is challenging.
+We will lower the entry barriers of Nibabel in particular, and more broadly, neuroimaging
+processing and analysis in general.
```

Comment:
Feels like this jumps in with the assumption that Nibabel is a known quantity. Rewrote slightly to give context, stealing some words from below.

```suggestion
Nurturing active and diverse communities around core libraries is challenging.
Here we propose a two-fold strategy to lower barriers to entry for Nibabel, a
brain image workbench that forms a key component of neuroimaging in Python.

```